test/webgl-independent-tests.js

```diff
@@ -0,0 +1,10 @@
+// Imports tests for all modules that do not depend on WebGL
+
+// Shadertools is a GLSL textual processing library, no actual WebGL dependencies
+import '@luma.gl/shadertools/../test';
```

Comment:
This is not going to work per the current set up, see https://github.com/uber-web/loaders.gl/pull/115#discussion_r263826237

Reply:
> This is not going to work per the current set up, see uber-web/loaders.gl#115 (comment)

Updated to work with `dist` testing.